src/main/java/rx/exceptions/Exceptions.java

```diff
@@ -31,7 +31,7 @@
 
     /** Utility class, no instances. */
     private Exceptions() {
-        throw new IllegalStateException("No instances!");
+        throw new AssertionError("No instances.");
```

Comment:
I find it that this exception matches the use case a bit better since we're asserting that this constructor shall never be called.